src/main/java/seedu/club/model/ModelManager.java

```diff
@@ -209,6 +209,7 @@ public void removeGroup(Group toRemove) throws GroupNotFoundException, GroupCann
         clubBook.removeGroup(toRemove);
         indicateClubBookChanged();
     }
+    //@@author
```

Comment:
I don't think we should put empty @@author tags if the author of the next part is indicated.
At least, according to the example on the CS2103 website, we shouldn't.
The empty author tag is added if you don't know who the author of the code chunk is

Reply:
understood 👍